SUPPORTED_APIS.md

```diff
@@ -14,6 +14,14 @@ The `status` options are:
 
 | Namespace | API | <div style="width:130px">Status</div> | Description |
 | --- | --- | --- | --- |
+| `ANVIL` | `anvil_setNonce` | `SUPPORTED` | Sets the nonce of a given account |
+| `ANVIL` | `anvil_impersonateAccount` | `SUPPORTED` | Impersonate an account |
```

Comment:
```suggestion
| `ANVIL` | `anvil_impersonateAccount` | `SUPPORTED` | Send transactions impersonating an externally owned account or contract |
```

Reply:
I don't think impersonating sends any transactions - it just makrs the address...